prime-router/metadata/fhir_mapping/hl7/datatype/Annotation.yml

```diff
@@ -54,7 +54,20 @@ extension_1-noteSource:
       expressionType: HL7Spec
       type: STRING
 
-extension_2-noteType:
+extension_2-noteComment:
```

Comment:
Is there a mapping for NTE to Annotation? I don't see it in the inventory

Reply:
Not really, it seems to be based in [NTE[ServiceRequest]](https://docs.google.com/spreadsheets/d/1ycH9LSI0xRyZqiRmh233wdpmJ-MyjzPFe4015X2TDDY/edit#gid=0), that could probably be revisited, but for the purposes of this story, just decided to try to fit into the existing files.

Reply:
Yea, there is no official mapping for some reason, so this is just what made sense at the time